lib/StaticTensor.chpl

```diff
@@ -333,6 +333,7 @@ proc type staticTensor.fromShape(type eltType = real,shape: int...?rank,value: e
     const v = value;
     const dom = util.domainFromShape((...shape));
     const A: [dom] eltType;
+    A = value;
     var t = new staticTensor(A);
     return t;
```

Comment:
why does `const v = value` exist here? If I assume its for localization, then this should be `A = v;`